pkg/operator/client/client.go

```diff
@@ -0,0 +1,15 @@
+package client
+
+import (
+	pb "github.com/dapr/dapr/pkg/proto/operator"
+	"google.golang.org/grpc"
+)
+
+// GetOperatorClient returns a new k8s operator client and the underlying connection
+func GetOperatorClient(address string) (pb.OperatorClient, *grpc.ClientConn, error) {
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
```

Comment:
Please add stats handler to this client to track the failure of the grpc request like below.

https://github.com/dapr/dapr/blob/d8aa4008346735aa8e4746626c0a7a5eb4e9a9fa/pkg/runtime/security/auth.go#L86